src/components/common/Extensions/NotificationBanner/NotificationBanner.tsx

```diff
@@ -29,7 +29,7 @@ const NotificationBanner: FC<PropsWithChildren<NotificationBannerProps>> = ({
     <div
       className={clsx(
         styles.banner,
-        `text-gray-900 ${
+        `text-gray-900 gap-2 ${
```

Comment:
I think this color isn't needed because it's set by default in global.css`text-gray-900`